src/SimpleWebRequest.ts

```diff
@@ -663,8 +662,11 @@ export class SimpleWebRequest<TBody, TOptions extends WebRequestOptions = WebReq
 
     private _deferred: SyncTasks.Deferred<WebResponse<TBody, TOptions>>;
 
-    constructor(action: string, url: string, options: TOptions, getHeaders?: () => Headers,
-            blockRequestUntil?: () => SyncTasks.Promise<void>|undefined) {
+    constructor(action: string,
+                url: string,
+                options: TOptions,
+                getHeaders?: () => Headers,
+                blockRequestUntil?: () => SyncTasks.Promise<void> | undefined) {
```

Comment:
This uses `SyncTasks.Promise` but other places use `TSPromise` or SyncTasks.STPromise`. All these files should be consistent.

Reply:
[`Promise` is alias for `STPromise`](https://github.com/microsoft/SyncTasks/blob/master/src/SyncTasks.ts#L153), anyway replaced .